indico/MaKaC/participant.py

```diff
@@ -448,16 +448,8 @@ def addPendingParticipant(self, participant):
 
             profileURL = urlHandlers.UHConfModifParticipantsPending.getURL(self._conference)
 
-            toList = []
-            creator=self._conference.getCreator()
-            if isinstance(creator, AvatarUserWrapper):
-                toList.append(creator.getEmail())
-            for manager in self._conference.getAccessController().getModifierList() :
-                if isinstance(manager, AvatarUserWrapper):
-                    toList.append(manager.getEmail())
-
             data = {}
-            data["toList"] = toList
+            data["toList"] = self._conference.all_manager_emails
```

Comment:
Yeah, as you see I've deleted the comment almost immediately, as I realised that.